On auth expire soon event (#545)
SAP/luigi: docs/authorization-events.md

    Return `false` to prevent redirecting to `logoutUrl` after executing this function. It goes to the Luigi main route `/` instead.
-   `onAuthExpired` **[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed if the token expires during runtime, or if Luigi is opened with outdated authorization data in the local storage. Return `false` to prevent redirecting to `logoutUrl` after executing this function.
-   `onLogout` **[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed after the user logs out.

> - `onAuthExpireSoon`**[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed before token will expire. For oAuth2 and OIDC it is possible to specify the seconds when the event will be fired by setting **accessTokenExpiringNotificationTime**.

Reviewer: ```suggestion
- `onAuthExpireSoon`**[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed before the token expires. For oAuth2 and OIDC you can set the **accessTokenExpiringNotificationTime** to specify the number of seconds required to pass before the notification is fired.
```

    Return `false` to prevent redirecting to `logoutUrl` after executing this function. It goes to the Luigi main route `/` instead.
-   `onAuthExpired` **[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed if the token expires during runtime, or if Luigi is opened with outdated authorization data in the local storage. Return `false` to prevent redirecting to `logoutUrl` after executing this function.
-   `onLogout` **[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed after the user logs out.

> - `onAuthExpireSoon` **[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed before the token expires. For oAuth2 and OIDC you can set the **accessTokenExpiringNotificationTime** to specify the number of seconds required to pass before the event is fired. The default value is `60` seconds.

Reviewer: ```suggestion
- `onAuthExpireSoon` **[function](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/function)** is executed before the token expires. For OAuth2 and OIDC you can set the **accessTokenExpiringNotificationTime** to specify the number of seconds required to pass before the event is fired. The default value is `60` seconds.
```

>     - by using oAuth2ImplicitGrant you can specify **expirationCheckInterval**, which is the number of seconds to pass between each check if the token is about to expire. The default value is `5` seconds.

Reviewer: ```suggestion
    - by using oAuth2ImplicitGrant you can specify **expirationCheckInterval** which is the number of seconds to pass between each check if the token is about to expire. The default value is `5` seconds.
```

>     - by using oAuth2ImplicitGrant you can specify **expirationCheckInterval**, which are the number of seconds to pass between each check if the token is about to expire. The default value is `5` seconds.

Reviewer 2: ```suggestion
    - by using oAuth2ImplicitGrant you can specify **expirationCheckInterval**, which is the number of seconds to pass between each check if the token is about to expire. The default value is `5` seconds.
```
